altos-rust/altos-core/libs/heap/free_list_allocator/src/free_list.rs

```diff
@@ -153,7 +155,7 @@ impl FreeList {
     // Traverses the free list, looking for two sequential free blocks which return true
     // when passed into match_condition.
     fn find_block<F: Fn(Option<&BlockHeader>, &BlockHeader) -> bool>(&mut self, match_condition: F)
-        -> (Option<&'static mut BlockHeader>, Option<&'static mut BlockHeader>) {
+    -> (Option<&'static mut BlockHeader>, Option<&'static mut BlockHeader>) {
```

Comment:
Same as above. I'm not sure what the right way to indent this would be, but it looks kind of funny to me having it on the same level of indentation as `fn find_block`.